### PR TITLE
move all integration & verify job to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
also renamed ci-testgo to integration
also make them use stable1,2,3 syntax so for next releases we don't have to delete/recreate them

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -11710,6 +11710,129 @@ periodics:
       - name: GOPATH
         value: /go
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180202-1036088f2-1.8
+- interval: 1h
+  name: ci-kubernetes-integration-master
```
Comment (BenTheElder) [inline prow/config.yaml]: missing agent

Comment (BenTheElder) [reply prow/config.yaml]: ditto for other jobs below

**Comment by rmmh** [comment]
I prefer -1.7 -1.8 to -stable1 -stable2 etc. Renaming something each release is not that big of a deal, and makes it easier for people looking at branch results.

**Comment by rmmh** [comment]
Also, -stable1 cutting over from 1.7 to 1.8 is confusing!

**Comment by BenTheElder** [comment]
> I prefer -1.7 -1.8 to -stable1 -stable2 etc. Renaming something each release is not that big of a deal, and makes it easier for people looking at branch results.

It was a big deal when it involved > 1MB of config across at least a half dozen files. It should be less of a big deal now, but I think we've already mostly migrated this way?

**Comment by krzyzacy** [comment]
in testgrid it will still display as release versions though
what's cutting over 1.7 to 1.8?

**Comment by rmmh** [comment]
Would a later commit make stable1 target 1.10 and stable2 target 1.9? That's a little odd. I'd want buy-in from release managers.

**Comment by krzyzacy** [comment]
(I needed to rename it from `release-1.7` to `release-1-7` anyway, might as well as unify it with other job names)

so this has been like this for past two release - has some doc around https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs

what will happen is:
after each new release branch is cut, we will put the new release into the beta channel, aka for 1.10 it will be
k8s-dev : master
k8s-beta : release-1.10
k8s-stable1 : release-1.9
k8s-stable2 : release-1.8
k8s-stable3 : release-1.7

and after 1.10 is cut and 1.7 tests goes to stale, we do it again
k8s-dev : master
k8s-beta : master (inactive)
k8s-stable1 : release-1.10
k8s-stable2 : release-1.9
k8s-stable3 : release-1.8

there are still some manual process like fix testgrid config, fix some release branch in `--repo` flags, but we won't need to delete all the jobs in the end of a release, and recreate them at the beginning of a release.

**Comment by krzyzacy** [comment]
I'll move the ci build jobs next, and maybe bikeshed names & migrate context from pull-kubernetes-unit -> pull-kubernetes-integration tomorrow

**Comment by krzyzacy** [comment]
/hold cancel
:eyes:

**Comment by krzyzacy** [comment]
mungegithub :hocho: 
/retest

**Comment by krzyzacy** [comment]
@cjwagner ^^ or https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/6653/pull-test-infra-bazel/12827/
I think this is not related to my change? as 1st test passes
/shrug
/retest

**Comment by cjwagner** [comment]
@krzyzacy It appears to be a flakey submit queue test which I have no intention of fixing.  🌊 🌊 🌊

**Pull request merged**